pages/index.js

```diff
@@ -0,0 +1,187 @@
+import Head from 'next/head';
+import Error from 'next/error';
+import cn from 'classnames';
+import styles from '../styles/Home.module.css';
+import { getRecord } from '../utils';
+import MenuItem from '../components/MenuItem';
+import Modifier from '../components/Modifier';
+import ColdbrewMenuItems from '../components/ColdbrewMenuItem';
+import { useEffect } from 'react';
+import { useRouter } from 'next/router';
+
+const espressoAndCoffeeGroupTag = 'espresso-and-coffee';
+const icedColdbrewGroupTag = 'iced-cold-brew';
+const menuApi = 'https://menu-api.raydiant.com/v1/groups';
+const apiKey = process.env.RAYDIANT_MENU_API_KEY ?? '';
+
+export const getServerSideProps = async (context) => {
+  const {
+    menu,
+    footnote = '',
+    espressoAndCoffeeSubheading = '',
+    espressoAndCoffeeBottomText = '',
+  } = context.query;
+
+  // Set the response status code to BadRequest if missing the menu query param.
+  if (!menu) {
+    context.res.statusCode = 400;
+    return {
+      props: {
+        errorCode: context.res.statusCode,
+        errorTitle: 'Please provide a menu',
+      },
+    };
+  }
+
+  // Make request to the On-Brand Menu API.
+  const res = await fetch(`${menuApi}?menus=${menu}&depth=5`, {
+    headers: { 'X-API-Key': apiKey },
+  });
+  // Forward the response status code from the On-Brand Menu API.
+  if (!res.ok) {
+    context.res.statusCode = res.status;
+    return {
+      props: {
+        errorCode: context.res.statusCode,
+        errorTitle: `Failed to load menu for ${menu}`,
+      },
+    };
+  }
+
+  const data = await res.json();
+
+  return {
+    props: {
+      data,
+      footnote,
+      espressoAndCoffeeSubheading,
+      espressoAndCoffeeBottomText,
+    },
+  };
+};
+
+export default function Home({
+  errorCode,
+  errorTitle,
+  data,
+  footnote,
+  espressoAndCoffeeSubheading,
+  espressoAndCoffeeBottomText,
+}) {
+  const router = useRouter();
+
+  useEffect(() => {
+    const refreshData = () => {
+      router.replace(router.asPath);
+    };
+
+    const id = setInterval(() => {
+      refreshData();
+    }, 30000);
```

Comment:
Just confirming, was this 30s in the original starbucks app? Looks like it's 30s in the whole foods app but wonder if we should set it to 10s to make demos a little nicer. No preference on my end really, up to you!

Reply:
My mistake! I'll update this to 10 seconds. Thanks for picking that up 👍